rules/aws_cloudtrail_rules/aws_ipset_modified.py

```diff
@@ -14,7 +14,7 @@ def rule(event):
 
 
 def title(event):
-    return "IPSet was modified in " f"[{event.get('recepientAccountId','')}]"
+    return "IPSet was modified in " f"[{event.get('recipientAccountId','')}]"
```

Comment:
Looking at the sample logs, `recipientAccountId` may not be a valid value --
```
awsregion: us-east-1
eventid: abc-123
eventname: CreateIPSet
eventsource: guardduty.amazonaws.com
eventtime: "2022-07-17 04:50:23"
eventtype: AwsApiCall
eventversion: "1.08"
...
```

Perhaps combining `awsregion` and `eventid` would be a suitable substitute? cc: @edyesed